src/styles/styles.js

```diff
@@ -228,23 +228,18 @@ const styles = {
     pill: {
         borderRadius: 14,
         backgroundColor: themeColors.pillBG,
-        height: variables.componentSizeSmall,
+        height: 20,
         flexDirection: 'row',
-        paddingTop: 6,
-        paddingBottom: 6,
         paddingLeft: 7,
         paddingRight: 7,
```

Comment:
A couple of minor changes to make [according to the Figma design](https://www.figma.com/file/hXumnAqZUEULJcP8SFnvYT/NL5%3A-Request-Money-%26-Split-Bills?node-id=128%3A0):
- BadgeHeight should be 26
- HorizontalPadding should be 8